cypress/e2e/tests/migration/migration-waves/crud.test.ts

```diff
@@ -149,7 +149,7 @@ describe(["@tier0"], "Migration Waves CRUD operations", () => {
 
     after("Clear test data", function () {
         deleteAllStakeholders();
```

Comment:
The PR looks good to me . 

I do have a question though . I'm wondering why deleteAllStakeholders(); is being used instead of deleteByList(stakeholdersList); for clean up.

Reply:
`deleteAllStakeHolder` can be used as well as `deleteAllStackHolderGroups` could be used before.

The thing is that one of the selectors of the `StakeHolderGroups` table has changed and now the deleteAll method doesn't work for groups, but it still does for stakeHolders.